src/Storages/System/StorageSystemServerSettings.cpp

```diff
@@ -28,7 +47,8 @@ NamesAndTypesList StorageSystemServerSettings::getNamesAndTypes()
         {"description", std::make_shared<DataTypeString>()},
         {"type", std::make_shared<DataTypeString>()},
         {"is_obsolete", std::make_shared<DataTypeUInt8>()},
-        {"is_hot_reloadable", std::make_shared<DataTypeUInt8>()}
+        {"is_hot_reloadable", std::make_shared<DataTypeUInt8>()},
+        {"runtime_reload", std::make_shared<DataTypeEnum8>(getTypeEnumsAndValues())}
```

Comment:
There is no restriction that new columns must be *appended*, we can insert them at arbitrary positions and even permute the order of existing columns.

Thoughts:
- `is_obsolete` should be the last column
- `is_hot_reloadable` and `runtime_reload` are related columns and should therefore be adjacent to each other
- let's move the two former columns prior to `is_obsolete`.

Reply:
`runtime_reload` isn't a great name (`is_hot_reloadable` isn't either but it is too late to change it, I guess).

So if I understand correctly, 'is_hot_reloadable` shows *if* a server setting can be changed and `runtime_reload` shows how it can be changed. There is some duplication here. I think the latter is conceptually a full replacement of the former, right?

Reply:
@rschu1ze I think perhaps the value of "is_hot_reloadable" is problematic. It seems that only those that undergo changes (where the set value is inconsistent with the displayed value) will be shown as true.

Reply:
https://github.com/ClickHouse/ClickHouse/blob/20c0347eefa55b50f03a2e4076bd9bcf771cc2c6/src/Storages/System/StorageSystemServerSettings.cpp#L38-L61

https://github.com/ClickHouse/ClickHouse/blob/20c0347eefa55b50f03a2e4076bd9bcf771cc2c6/src/Storages/System/StorageSystemServerSettings.cpp#L79

Reply:
@rschu1ze Hi, Can I delete "is_hot_reloadable" and change "runtime_reload" to "hot_reload"?